src/main/io/ledstrip.c

```diff
@@ -1321,23 +1321,23 @@ bool parseColor(int index, const char *colorConfig)
 /*
  * Redefine a color in a mode.
  * */
-bool setModeColor(ledModeIndex_e modeIndex, int modeColorIndex, int colorIndex)
+bool setModeColor(ledFCStates_e fcStatusIndex, int modeColorIndex, int colorIndex)
 {
     // check color
     if (colorIndex < 0 || colorIndex >= LED_CONFIGURABLE_COLOR_COUNT)
         return false;
-    if (modeIndex < LED_MODE_COUNT) {  // modeIndex_e is unsigned, so one-sided test is enough
+    if (fcStatusIndex < LED_FCSTATE_COUNT) {  // ledFCStates_e is unsigned, so one-sided test is enough
         if (modeColorIndex < 0 || modeColorIndex >= LED_DIRECTION_COUNT)
             return false;
-        ledStripStatusModeConfigMutable()->modeColors[modeIndex].color[modeColorIndex] = colorIndex;
-    } else if (modeIndex == LED_SPECIAL) {
-        if (modeColorIndex < 0 || modeColorIndex >= LED_SPECIAL_COLOR_COUNT)
+        ledStripDetailedModeConfigMutable()->modeColors[fcStatusIndex].color[modeColorIndex] = colorIndex;
+    } else if (fcStatusIndex == LED_FCSTATE_SPECIAL) {
+        if (modeColorIndex < 0 || modeColorIndex >= LED_SPECIAL_FCSTATE_COLOR_COUNT)
             return false;
-        ledStripStatusModeConfigMutable()->specialColors.color[modeColorIndex] = colorIndex;
-    } else if (modeIndex == LED_AUX_CHANNEL) {
+        ledStripDetailedModeConfigMutable()->specialColors.color[modeColorIndex] = colorIndex;
+    } else if (fcStatusIndex == LED_FCSTATE_AUX_CHANNEL) {
         if (modeColorIndex < 0 || modeColorIndex >= 1)
             return false;
-        ledStripStatusModeConfigMutable()->ledstrip_aux_channel = colorIndex;
+        ledStripDetailedModeConfigMutable()->ledstrip_aux_channel = colorIndex;
```

Comment:
`ledstrip_aux_channel` is RC channel, but colorIndex in stored into it?!

Reply:
yes that's rather strange... I have no clue, to be honest.

Reply:
I'll look into it

Reply:
Thanks!